### DISCUSSION TITLE
Is it possible to train PGNet with special characters?

### DISCUSSION THREAD

**Discussion by SaiKrishnaBala** [PaddlePaddle/PaddleOCR#6836] [Q&A]
Is there any limit to the number of characters used in the dictionary for PGNet?

**Comment by LDOUBLEV** [comment]
YES! The dictionary file needs to be replaced when training other characters.  The default is English 26 characters

**Reply by SaiKrishnaBala** [reply]
I have replaced the dictionary file by including alphabets, numbers and digits, which came upto 94. Do I need to change the pad_num field in the configuration? I tried changing it but I got the below error:
`ValueError: (InvalidArgument) The value of Attr(blank) should be in interval [0, 94), but received
[Hint: Expected blank < sequence_width, but received blank:94 >=....`
